server/src/main/kotlin/com/localove/entities/Person.kt

```diff
@@ -0,0 +1,60 @@
+package com.localove.entities
+
+import com.localove.Identifiable
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.stereotype.Repository
+import java.time.LocalDate
+import javax.persistence.*
+
+@Entity
+@Table(name = "users")
+class Person(
+
+    @Column(name = "login")
+    val login: String,
+
+    @Column(name = "name")
+    val name: String,
+
+    @Column(name = "gender")
+    @Enumerated(EnumType.STRING)
+    val gender: Gender,
+
+    @Column(name = "birth_date")
+    val birthDate: LocalDate,
+
+    @Column(name = "status")
+    val status: String,
```

Comment:
```suggestion
    var status: String? = null,
```
нам же нужно будет через данный класс менять статус, почту, bio и т.д., а с `val` это ж нельзя сделать

Reply:
Сделал все var, кроме gender (долго спорили, можно ли менять его или нет))) )
Ну и да, изменяемые поля лучше не nullable делать, ибо столбцы соответствующие в бд не nullable. да и вообще зачем лишние заморочки с нуллами... bio и status сделал с дефолтными значениями - пустыми строками.